packages/plugin-sequelize/src/utils/public.ts

```diff
@@ -195,12 +201,14 @@ export function getFieldIncludeOptions(options: {
     includeOptions.order = orderOptions
   }
 
-  const argPage = typeof options.args.page === 'number' ? options.args.page : PAGE_ARG_DEFAULT
-  const argPerPage =
-    typeof options.args.perPage === 'number' ? options.args.perPage : PER_PAGE_ARG_DEFAULT
+  if (options.isList) {
+    const argPage = typeof options.args.page === 'number' ? options.args.page : PAGE_ARG_DEFAULT
+    const argPerPage =
+      typeof options.args.perPage === 'number' ? options.args.perPage : PER_PAGE_ARG_DEFAULT
 
-  includeOptions.offset = (argPage - 1) * argPerPage
-  includeOptions.limit = argPerPage
+    includeOptions.offset = (argPage - 1) * argPerPage
+    includeOptions.limit = argPerPage
+  }
```

Comment:
This is simply wrapping the `offset` and `limit` with a condition `if (options.isList)`. It fixes an issue where Sequelize was throwing an error stating that those properties are only supported on `HasMany` associations.